<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CleaverWidget</class>
 <widget class="QDockWidget" name="CleaverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cleaving Tool</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Basic</string>
     </attribute>
     <widget class="QToolButton" name="createMeshButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>130</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Constuct Mesh</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="noneRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="subdivideRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Octree Subdivision</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="cleaveRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Topological Cleaving</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="topologyLabel">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Topology Correction</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Advanced</string>
     </attribute>
     <widget class="QToolButton" name="stencilTetsButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>146</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stencil Tets</string>
      </property>
     </widget>
     <widget class="QToolButton" name="buildAdjacencyButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>26</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Build Adjacency</string>
      </property>
     </widget>
     <widget class="QToolButton" name="sampleVolumeButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>46</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample Volume</string>
      </property>
     </widget>
     <widget class="QToolButton" name="computeInterfacesButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>86</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute Interfaces</string>
      </property>
     </widget>
     <widget class="QToolButton" name="snapAndWarpButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>126</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Snap and Warp</string>
      </property>
     </widget>
     <widget class="QToolButton" name="computeAlphasButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>66</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute Alphas</string>
      </property>
     </widget>
     <widget class="QToolButton" name="generalizeTetsButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>106</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Generalize Tets</string>
      </property>
     </widget>
     <widget class="QToolButton" name="createBackgroundMeshButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>6</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Background Mesh</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>createBackgroundMeshButton</tabstop>
  <tabstop>sampleVolumeButton</tabstop>
  <tabstop>computeInterfacesButton</tabstop>
  <tabstop>generalizeTetsButton</tabstop>
  <tabstop>snapAndWarpButton</tabstop>
  <tabstop>stencilTetsButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>createBackgroundMeshButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>createBackgroundMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampleVolumeButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>sampleVolume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>computeInterfacesButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>computeInterfaces()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalizeTetsButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>generalizeTets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stencilTetsButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>stencilTets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>snapAndWarpButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>snapAndWarp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildAdjacencyButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>buildMeshAdjacency()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>computeAlphasButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>computeAlphas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createMeshButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>createMesh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noneRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>topologyActionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subdivideRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>topologyActionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleaveRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>CleaverWidget</receiver>
   <slot>topologyActionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createBackgroundMesh()</slot>
  <slot>computeInterfaces()</slot>
  <slot>sampleVolume()</slot>
  <slot>generalizeTets()</slot>
  <slot>stencilTets()</slot>
  <slot>snapAndWarp()</slot>
  <slot>computeAlphas()</slot>
  <slot>buildMeshAdjacency()</slot>
  <slot>createMesh()</slot>
  <slot>topologyActionChanged()</slot>
 </slots>
</ui>
